README.md

```diff
@@ -23,16 +26,33 @@ Terra Toolkit is a utility module used to facilitate independent development of
 
 - Install with [npm](https://www.npmjs.com): `npm install terra-toolkit --save-dev`
 
-Terra Toolkit uses docker to setup, run, and tear down selenium to ensure a consistent testing environment locally and in continuous integration build systems. To use Terra Toolkit you must install docker on your machine. Installation instructions can be found at https://www.docker.com/.  **Requires Docker v17.09.0 or higher.**
+Terra Toolkit uses Docker to setup, run, and tear down selenium to ensure a consistent testing environment locally and in continuous integration build systems. To use Terra Toolkit for webdriver testing, you must install docker on your machine.
 
+- Install Docker version 17.09.0 or higher. Installation instructions can be found at https://www.docker.com/.
```

Comment:
Docker has a nice installation guide: https://docs.docker.com/install/

It's kind of hard to find from the home page 👎